HelpSource/Help.schelp

```diff
@@ -2,38 +2,266 @@ TITLE::Help
+link::Guides/UsingMIDI##MIDI::,
+link::Guides/OSC_communication##OSC:: and
+link::Guides/Working_with_HID##HID::,
+sclang can be easily interfaced with local and networked hardware and software.
+
+## soft:: ::• soft:: ::link::Classes/ScIDE:: soft:: ::– a dedicated editor for SuperCollider with an integrated help system.||
+link::Guides/SCIde##The SuperCollider-IDE (Integrated Development Environment) :: allows you to start writing code right out of the box. Community members also maintain sclang integration with other popular IDEs such as link::https://github.com/scztt/vscode-supercollider##Visual Studio Code, VSCodium::, link::https://github.com/capital-G/sc_kernel##Jupyter::, link::https://github.com/davidgranstrom/scnvim##SCNvim::, link::https://github.com/supercollider/scvim##scvim::, link::https://github.com/supercollider/scel##Emacs::, and so on.
+::
+
+link::Guides/ClientVsServer##The server and client:: communicate via link::http://opensoundcontrol.org/##OSC (open sound control):: messages over link::Guides/Glossary#kw_udp#UDP::, allowing SuperCollider to run on a single machine or link::Guides/MultiClient_Setups##on multiple machines over a network::. Users can also diretly control the sc-server using link::Guides/NodeMessaging::.
+
+If you wish, you can extend SuperCollider's functionality by installing optional link::https://github.com/supercollider/sc3-plugins##sc3-plugins:: distributed by the SC development team, and by installing individual extensions from the custom libraries called link::Classes/Quarks:: developed by individual users. For more information, see the link::https://github.com/supercollider-quarks/quarks##Directory of community contributed Quarks for SuperCollider:: and the link::https://quarks.supercollider.online/##SuperCollider Quarks Web Index::.
+
+table::
+## * SuperCollider link::https://www.audiosynth.com##was originally developed:: by James McCartney and link::https://supercollider.github.io/##is now distributed:: as Free Software under the GNU General Public License. It link::https://github.com/supercollider/supercollider##is maintained and developed :: by an active and enthusiastic community.
 ::
 
+table::
+##** SuperCollider can be installed and run on link::https://supercollider.github.io/downloads.html#linux#Linux::,
+link::https://supercollider.github.io/downloads.html#mac#macOS:: and
+link::https://supercollider.github.io/downloads.html#windows#Windows::,
+as well as microcomputers such as
+link::https://github.com/supercollider/supercollider/blob/develop/README_RASPBERRY_PI.md##Raspberry Pie:: and
+link::https://github.com/supercollider/supercollider/blob/develop/README_BEAGLEBONE_BLACK.md##BeagleBone Black::.
+link::https://www.arduino.cc/##Arduino:: or equivalent devices can be
+controlled via link::Classes/SerialPort:: within sclang.
+DMX can also be used by means of user-developed libraries (link::https://quarks.supercollider.online/##type dmx in the search box on the linked web page::
```

Comment:
I don't think that Arduino and DMX should be mentioned in a footnote about _cross-platform_ capabilities.

I suggested adding SerialPort to the protocol list in the Client section. Note sure where/if to mention Arduino and DMX elsewhere, or if those are best left to other docs.

Reply:
Also, I'm not sure about using the table format for local footnotes. Just using the footnotes tag may be better, even though it will put them at the bottom of the page. 
I could see the argument for the supported platforms to be listed as another paragraph in this introduction, rather than a footnote.

Reply:
I removed the sentences mentioning Arduino and DMX. I also removed tables for local footnotes and inserted those paragraphs into the first paragraph of the introduction.